wafer/talks/tests/test_views.py

```diff
@@ -353,22 +355,40 @@ def test_view_seven_speakers(self):
         self.check_n_speakers(7, [(0, 4), (4, 7)])
 
 
-class TalkViewSetTests(TestCase):
+class SortedResultsClient(APIClient):
+    def __init__(self, *args, **kw):
+        self._sort_key = kw.pop('sort_key')
+        super(SortedResultsClient, self).__init__(*args, **kw)
+
+    def _sorted_response(self, response):
+        def get_key(item):
+            return item[self._sort_key]
+        if response.data and 'results' in response.data:
+            response.data['results'].sort(key=get_key)
+        return response
+
+    def generic(self, *args, **kw):
+        response = super(SortedResultsClient, self).generic(*args, **kw)
+        return self._sorted_response(response)
 
```

Comment:
This isn't terrible specific to talks - should we move it to a test_utils module so it can be used when testing other apis?